src/changes.apib

```diff
@@ -48,6 +48,11 @@ _This will only affect the version for those API calls and won't affect any othe
 
 We list all backwards-incompatible changes here. As described above, new additions and forwards-compatible changes don’t need a new API version and will not appear in this list.
 
+#### Beta
+
+- The property `status` in `invoices.list` and `invoices.info` now returns `outstanding` and `matched` instead of `booked`.
+- The `filter` `status` property in the `invoices.list` request now only accepts an `array` of values.
```

Comment:
B : the filter also doesn't allow to send it "booked" as a status anymore... should we tell customers what to do instead? ("send `["oustanding", "matched"]` to keep filtering on "booked"?)